src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -470,11 +478,33 @@ public boolean isEnqueued() {
     }
 
     /**
-     * Clears this reference object and adds it to the queue with which
-     * it is registered, if any.
+     * Clears this reference object, then attempts to add it to the queue with
+     * which it is registered, if any.
+     *
+     * <p>If this reference is registered with a queue but not yet enqueued,
+     * the reference is added to the queue; this method is
+     * <b><i>successful</i></b> and returns true.
+     * If this reference is not registered with a queue, or was already enqueued
+     * (by the garbage collector, or a previous call to {@code enqueue}), this
+     * method is <b><i>unsuccessful</i></b> and returns false.
+     *
+     * <p><a href="package-summary.html#MemoryConsistency">Memory consistency effects</a>:
```

Comment:
Note the `<a href="package-summary.thml#Foo>` syntax can be replaced with something like
`@link java.lang.ref##Foo`.

Reply:
Thanks. I've updated the links that don't involve italicized/emphasized text.